src/components/sidebar/CallTreeSidebar.js

```diff
@@ -99,20 +110,45 @@ class ImplementationBreakdown extends React.PureComponent<
   }
 }
 
+type CategoryBreakdownProps = {|
+  +breakdown: BreakdownByCategory,
+  +categoryList: CategoryList,
+|};
+
+class CategoryBreakdown extends React.PureComponent<CategoryBreakdownProps> {
+  render() {
+    const { breakdown, categoryList } = this.props;
+    const data = breakdown.map((value, categoryIndex) => {
```

Comment:
I think this would be more valuable if it was sorted by percentage, with the heaviest categories at the top. It would make the list much more scannable.

Reply:
Good idea !

Reply:
If/when we have histograms we can revisit this. I also find it nice to have them in always the same order, but I agree that without histograms this makes it more difficult to look at.